readme.txt

```diff
@@ -97,6 +97,10 @@ We'd love your help! Here's a few things you can do:
 
 == Changelog ==
 
+= 3.1.39 =
+* Fix: 'Class "Parsedown" not found' by adding `erusev/parsedown` via PHP Scoper.
+* Chore: Data type check for loading scripts and styles.
```

Comment:
* Chore: Upgraded NPM packages resolving security issues present with trim-newlines library.
* Fix: Replaced Moment with [Dayjs](https://day.js.org/), it helps in speed and tackles security issues with the Moment library.